src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -359,6 +359,14 @@ export class NotebookModel extends Disposable implements INotebookModel {
 				if (contents.cells && contents.cells.length > 0) {
 					this._cells = contents.cells.map(c => {
 						let cellModel = factory.createCell(c, { notebook: this, isTrusted: isTrusted });
+						/*
+						In a parameterized notebook there will be an injected parameter cell.
+						We need to indicate to the user the difference between this cell and the parameters cell.
```

Comment:
Would be good to explain why this was chosen (link to papermill docs would be preferable)

Reply:
This was chosen since the papermill Injected-Parameter inputs "# Parameters" at the start of the cell always. I did this to indicate the difference between this cell and the Parameters cell. I can add the part where papermill originally injects the cell with the parameter with the comment "# Parameters" which would make it confusing to the user between the difference of that cell and the tagged parameters cell.

Reply:
I'm saying add that in a comment - people looking at this code later on aren't going to know why this particular format was chosen 😄

Reply:
Sure thing, just wanted to clarify if this was clear enough. 
`papermill originally inserts the injected parameter with the comment "# Parameters" which would make it confusing to the user between the difference of this cell and the tagged parameters cell. So to make it clear we edit the injected parameters comment to clearly indicate it is the Injected-Parameters cell`

Reply:
Looks good!